app/mono_pkg.yaml

```diff
@@ -15,6 +15,7 @@ stages:
     - test: --total-shards 7 --shard-index 4
     - test: --total-shards 7 --shard-index 5
     - test: --total-shards 7 --shard-index 6
+    - test:  -t presubmit-only --run-skipped
```

Comment:
Let's not run skipped, if something is skipped, it's skipped for a reason.

If want to filter enable/disable tests then make tags and let the test tags be disable by default and then explicitly enable the tags you want to run in CI.
See what I did for `fragile` in `dart_test.yaml`, see also:
https://github.com/dart-lang/test/blob/master/pkgs/test/doc/configuration.md#presets

It's not intuitive :see_no_evil:, but I think these tests can even be part of the sharded stuff, no need to create it's own thing...

```yaml
    - test: -P presubmit --total-shards 7 --shard-index 6
```

Then do `dart_test.yaml` as:
```yaml

  
tags:
  presubmit-only:
    skip: "Should only be run during presubmit"
    presets:
      presubmit:
        skip: false # Don't skip when running in presubmit
  fragile:
    skip: 'Only run fragile tests manually with `pub run test -P fragile`'
    presets:
      fragile:
        skip: false # Don't skip when running in -P fragile
```
I think that should work.

Reply:
Updated the presubmit preset. I wasn't aware of it, test configs can indeed be non-trivial :)